librz/asm/p/asm_rl78.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2023 Bastian Engel <bastian.engel00@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_types.h>
+#include <rz_util.h>
+#include <rz_lib.h>
+#include <rz_asm.h>
+#include "../arch/rl78/rl78.h"
+#include "../arch/rl78/instr.h"
+
+static int assemble(RzAsm *a, RzAsmOp *op, const char *buf) {
+        return 0x69;
+}
+
+static int disassemble(RzAsm *a, RzAsmOp *op, const ut8 *buf, int len) {
+        ut8 b[] = "\xca\x50\x69";
+        struct rl78_instr instr;
+        char c[64];
```

Comment:
Please rename `b` and `c` to something meaningful.

Reply:
This was for testing purposes and will be removed once proper testing has been implemented